#!/bin/sh
./manage.py runserver 41450 --insecure
